developer-workflow/communication-channels.rst

```diff
@@ -250,13 +250,26 @@ order to make open source pleasant for everyone involved.
 Additional Repositories
 =======================
 
-`Python Core Workflow`_ hosts the codebase for tools such as `cherry_picker`_
-and `blurb`_.
+`Python Core Workflow`_ hosts the codebase for tools such as `blurb`_.
+
+Other Core Workflows are:
+
+* `cherry_picker`_
```

Comment:
This section is about repos, so we should primarily link to the repo, but can also link to the PyPI page.

Reply:
You're not being dumb, it's good to ask questions :)

Maybe something like this?

* [cherry_picker](https://github.com/python/cherry-picker) ([PyPI](https://pypi.org/project/cherry_picker/))

Reply:
> You're not being dumb, it's good to ask questions :)
> 
> Maybe something like this?
> 
> * [cherry_picker](https://github.com/python/cherry-picker) ([PyPI](https://pypi.org/project/cherry_picker/))

Thank you for your co-operation, will make the changes mentioned above.